ceph.spec.in

```diff
@@ -563,9 +563,11 @@ Group:          System/Filesystems
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
 Requires:       python%{python3_pkgversion}-numpy
 Requires:       python3-scipy
+Requires:       python%{python3_pkgversion}-scikit-learn
 %if 0%{?rhel} == 7
 Requires:       numpy
 Requires:       scipy
+Requires:       scikit-learn
```

Comment:
i don't think el7 or EPEL7 has this package.

Reply:
yes, thanks @tchaikov I will add it at container side for rhel7, doesn't look like we can do much here.

Reply:
now that we're not building master on el7, does this matter?

Reply:
probably we can go further by removing the el7 stuff in `ceph.spec.in`.

Reply:
I was going to ask the same question, do we require it for previous ceph versions, or should we clean up el7 stuff?

Reply:
i'd suggest just drop the change for el7. and do the cleanup in a separate PR.

Reply:
ack. Thanks!